### PR TITLE
chore(changelog): add changelog entry about noIrregularWhitespace rule

### PR DESCRIPTION
**Pull request by michellocana**
## Summary
Update `CHANGELOG.md` with `noIrregularWhitespace` rule.

### CODE REVIEW

**Review by unvalley — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### Linter
 
+#### New features
+
+- Add [nursery/noIrregularWhitespace](https://biomejs.dev/linter/rules/no-irregular-whitespace). Contributed by @michellocana
+
+
```
Comment (unvalley) [inline CHANGELOG.md:51]: I think there are already `New Features section`, below Enhancements.

**Review by unvalley — CHANGES_REQUESTED** [review]
I left a comment

File: CHANGELOG.md (hunk 1 above)
Comment (Conaclos) [reply CHANGELOG.md:51]: There is also a `New rules` section. We should move `New rules` and `New features` before `Enhancements`. Also, we should merge `New rules` and `New features` into `New features` - this is what we usually do.

**Review by Conaclos — APPROVED** [review]
Thanks!

**Pull request merged**